airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/pipeline.py

```diff
@@ -753,7 +753,7 @@ async def run_connector_promote_pipeline(context: PublishConnectorContext, semap
                     all_modified_files.update(
                         await add_changelog_entry.export_modified_files(context.connector.local_connector_documentation_directory)
                     )
-                post_changelog_pr_update = CreateOrUpdatePullRequest(context, skip_ci=False, labels=["auto-merge"])
+                post_changelog_pr_update = CreateOrUpdatePullRequest(context, skip_ci=True, labels=["auto-merge", "auto-merge/promoted-rc"])
```

Comment:
Can `"auto-merge"` and `"auto-merge/promoted-rc"` be pulled in from the constants file?

Reply:
We could ideally add the `auto_merge` packages as a dependency of `pipelines` and import constant from it.

Reply:
Updated to add automerge as a dependency.